doc/toolbox-create.1.md

```diff
@@ -63,6 +67,25 @@ containers. Read more about the entry-point in `toolbox-init-container(1)`.
 
 ## OPTIONS ##
 
+**--authfile** AUTHFILE
+
+Path of the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json
+
+Note: You can also override the default path of the authentication file by
+setting the REGISTRY\_AUTH\_FILE environment variable. `export REGISTRY_AUTH_FILE=path`
+
+Note: This option is equal to one in `podman-create(1)`.
+
+**--creds** USERNAME:PASSWORD
```

Comment:
Ditto, as `--authfile`.

Generally, it's considered insecure to pass *secret* credentials through the command line because any other user on the system can look at the command lines of all processes on the system, including the credentials.

The better alternative is to feed such credentials through the standard input of the target program by somehow indirecting it. In the case of `podman pull`, I am not sure what would be the best approach. Worst case, we will have to use `podman pull --creds ...`, until we find a better way.

Regardless, let's not add this flag to the public interface of Toolbox unless someone comes up with a really good use-case for it.

Reply:
> Worst case, we will have to use podman pull --creds ..., until we find a better way.

I do not understand what you mean by this.

Reply:
Oh, my apologies about the rambling comment.

I wrote it before fully reading the entire pull request. I thought we are using `podman pull --creds username:password ...`, not `podman login`. So, I meant that if we really can't have a nice way to feed the credentials to Podman, then we can use `podman pull --creds ...` in whichever way that works until a better solution emerges.

In other words, I don't want Toolbox to encourage unsafe user behaviour (unless there's a compelling use-case, of course), but we have to work with the current realities of Podman. At least, folks can point the finger at us. :)

Reply:
I see that we are using `--creds` for testing. I wonder if we can solve that some other way.

Reply:
For the test suite, we should aim to make this work:
```
$ echo -e "username\npassword" | toolbox -y create --image registry.redhat.io/ubi8/ubi:8.4
```

That way we can still feed the toy credentials to the test suite without adding a `--creds` flag to our public user interface.

The other thing is to avoid passing the credentials, particularly the password, through the command line of the `podman login` invocation or any other spawned binary. So that a malicious user on the system can't sit there staring at the process list and steal the credentials.

In order to satisfy both constraints, we need to somehow get `podman login` to transparently pick up the credentials even when the standard input has been redirected to a non-terminal device, as will be the case for the test suite. Sadly, `podman login` insists on using the `--password-stdin` and `--username` flags when the input has been redirected, which makes it a bit annoying.

Therefore, I think `toolbox` should take care of reading the username and password itself in a way that works transparently with both terminal devices and otherwise. Then, we should use `podman login --password-stdin` to feed the password by putting it in a `bytes.Buffer` acting as the standard input.